atom/common/node_bindings.cc

```diff
@@ -165,7 +176,7 @@ void NodeBindings::Initialize() {
   node::Init(nullptr, nullptr, nullptr, nullptr);
 }
 
-node::Environment* NodeBindings::CreateEnvironment(
+node::Environment* NodeBindings::CreateRenderEnvironment(
```

Comment:
We can also use `AtomCommandLine` in renderer process so we don't have to duplicate the code.